tests/testapp/tests/test_signals.py

```diff
@@ -0,0 +1,29 @@
+import factory
+from django.test import TestCase
+from facility_profile.models import Facility
+from morango.models import DatabaseIDModel, DeletedModels, InstanceIDModel
+from morango.utils.controller import MorangoProfileController
+
+
+class FacilityModelFactory(factory.DjangoModelFactory):
+
+    class Meta:
+        model = Facility
+
+    name = factory.Sequence(lambda n: "Fac %d" % n)
+
+
+class PostDeleteSignalsTestCase(TestCase):
+
+    def setUp(self):
+        DatabaseIDModel.objects.create()
+        InstanceIDModel.get_or_create_current_instance()
+        [FacilityModelFactory() for _ in range(10)]
+        self.mc = MorangoProfileController('facilitydata')
+        self.mc._serialize_into_store()
+
+    def test_deleted_flag_gets_set(self):
+        facility = Facility.objects.first()
+        deleted_id = facility.id
+        facility.delete()
+        self.assertTrue(DeletedModels.objects.filter(id=deleted_id))
```

Comment:
Would be good to also add an explicit test case for CASCADE deletes, as I'm not actually sure how that interacts with signals (may be at a pure DB level). So, if you create a facility, then create a user in it, then delete the facility, is the user's ID in the `DeletedModels` table?